.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,42 @@
+---
+name: 🚀[Release] 
+about: An issue template to make sure the release process is followed
+title: '🚀[Release]'
+labels: 'Release'
+assignees: 'kcw-grunt'
+
+---
+
+## Goal 
+The process of publishing/launching a new release of **Litewallet** should be consistent and drama-free.  It should also allow the team to catch errors before they occur. This template should make it easy for any iOS developer in the team to be confident the release meets all our criteria.
+
+## Prepare code
+- [ ] 1. Locally create a new branch from `develop` with the pattern `release/v#.#.#` (Third dot should always be 0 which is preserved for [PATCHes](https://semver.org))
```

Comment:
@kcw-grunt so let's say we have a release called v2.2.0 and then we work on bug fixes only. According to `semver` the release has to be `release/v2.2.1` meaning `The third dot should always be 0` is always true.  Thoughts?

Reply:
Yes.....technically you are correct. It's just have we ever only done releases with bugfixes in the last 2 years?   I was thinking this template is for releases but let me address that.